src/contexts/TranslationContext.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { LANGS } from "../types/TranslationTypes";
+import defaultLang from "../i18n/en.json";
+
+export const TranslationContext = React.createContext({
```

Comment:
Could you please explain, why have you decided to lean on react context for maintaining locales instead of using store for those purposes?

_I'm not implying that you should only use store for that, I just want to hear your explanation_

Reply:
Redux store gives you the very same functionality